src/cascadia/TerminalSettingsEditor/SettingContainerStyle.xaml

```diff
@@ -21,37 +48,121 @@
         <Setter Property="Height" Value="19" />
         <Setter Property="Width" Value="19" />
         <Setter Property="Padding" Value="0" />
+        <Setter Property="BorderBrush" Value="Transparent" />
         <Setter Property="Background" Value="Transparent" />
     </Style>
 
     <Style x:Key="SettingContainerFontIconStyle"
            TargetType="FontIcon">
         <Setter Property="Foreground" Value="{StaticResource SystemAccentColor}" />
-        <Setter Property="FontSize" Value="14" />
-        <Setter Property="FontFamily" Value="Segoe MDL2 Assets" />
+        <Setter Property="FontSize" Value="11" />
+        <Setter Property="FontFamily" Value="Segoe Fluent Icons" />
+    </Style>
+
+    <Style x:Key="NonExpanderGrid"
+           TargetType="Grid">
+        <Setter Property="Background" Value="{ThemeResource ExpanderHeaderBackground}" />
+        <Setter Property="MinWidth" Value="{ThemeResource FlyoutThemeMinWidth}" />
+        <Setter Property="MinHeight" Value="64" />
+        <Setter Property="BorderThickness" Value="{ThemeResource ExpanderHeaderBorderThickness}" />
+        <Setter Property="BorderBrush" Value="{ThemeResource ExpanderHeaderBorderBrush}" />
+        <Setter Property="Padding" Value="16,0,8,0" />
+        <Setter Property="Margin" Value="0,4,0,0" />
+        <Setter Property="CornerRadius" Value="{ThemeResource ControlCornerRadius}" />
+    </Style>
+
+    <Style x:Key="SettingsPageItemHeaderStyle"
+           BasedOn="{StaticResource BodyTextBlockStyle}"
+           TargetType="TextBlock">
+        <Setter Property="LineHeight" Value="20" />
+        <Setter Property="TextWrapping" Value="WrapWholeWords" />
+    </Style>
+
+    <Style x:Key="SettingsPageItemDescriptionStyle"
+           BasedOn="{StaticResource CaptionTextBlockStyle}"
+           TargetType="TextBlock">
+        <Setter Property="Margin" Value="0,0,24,0" />
+        <Setter Property="LineHeight" Value="16" />
+        <Setter Property="Foreground" Value="{ThemeResource SubgroupHeaderBrush}" />
+        <Setter Property="TextWrapping" Value="WrapWholeWords" />
     </Style>
 
     <Style TargetType="local:SettingContainer">
-        <Setter Property="Margin" Value="0,24,0,0" />
         <Setter Property="IsTabStop" Value="False" />
+        <Setter Property="MaxWidth" Value="1000" />

```

Comment:
Is 1000 enough for super high resolution screens with the window maximized?

Reply:
I'm sure this value is in DIPs, So that should be 3000px at 300% scaling and thus fine.

Reply:
Aha OK